lib/tags/custom_tag/custom_tag.dart

```diff
@@ -73,17 +75,22 @@ class CustomTag extends PBTag implements PBInjectedIntermediate {
       layoutCrossAxisSizing: originalRef.layoutCrossAxisSizing,
       layoutMainAxisSizing: originalRef.layoutMainAxisSizing,
       isComponent: originalRef is PBSharedMasterNode &&
-          originalRef.componentSetName == null, //Variable used to add extra info. to custom file.
+          originalRef.componentSetName ==
+              null, //Variable used to add extra info. to custom file.
     );
   }
 }
 
 class CustomTagGenerator extends PBGenerator {
   /// Variable that dictates in what directory the tag will be generated.
-  static const DIRECTORY_GEN = 'controller/tag';
+  static String DIRECTORY_GEN = GetIt.I.get<PathService>().widgetsRelativePath;
+  static String DIRECTORY_CUSTOM =
+      GetIt.I.get<PathService>().customRelativePath;
 
   @override
   String generate(PBIntermediateNode source, PBContext context) {
+    var customDirectory =
+        DIRECTORY_GEN + '/' + context.tree.name + '/' + DIRECTORY_CUSTOM;
```

Comment:
For this part, it may be better to use the path.join() method to have more reliable paths

Reply:
You are correct, let me do this change